sympy/solvers/ode.py

```diff
@@ -303,7 +303,9 @@
     "nth_linear_constant_coeff_variation_of_parameters",
     "nth_linear_euler_eq_nonhomogeneous_variation_of_parameters",
     "Liouville",
+    "2nd_linear_airy",
     "2nd_power_series_ordinary",
+    "Bessel_Equation",
```

Comment:
Better to name new hints consistently with existing names.  I suggest "2nd_linear_bessel"